src/__tests__/__snapshots__/duplicated-names.spec.ts.snap

```diff
@@ -13,10 +13,9 @@ declare export var AuthMechanism: {
   +MONGODB_CR: \\"MONGODB-CR\\",
   ...
 };
-export type AuthMechanismType1 = $ElementType<
-  typeof AuthMechanism,
-  $Keys<typeof AuthMechanism>
->;
+export type AuthMechanismType1 = (typeof AuthMechanism)[$Keys<
```

Comment:
Don't know what is adding parenthesis around `(typeof AuthMechanism)` but it's not breaking types 🤷‍♂️